features/fixtures/news_and_communications.json

```diff
@@ -0,0 +1,113 @@
+{
+  "analytics_identifier":null,
+  "base_path":"/news-and-communications",
+  "content_id":"b7574bba-969f-4c49-855a-ae1586258ff6",
+  "content_purpose_document_supertype":"navigation",
+  "document_type":"finder",
+  "email_document_supertype":"other",
+  "first_published_at":"2018-11-07T14:47:48.000+00:00",
+  "government_document_supertype":"other",
+  "locale":"en",
+  "navigation_document_supertype":"other",
+  "phase":"alpha",
+  "public_updated_at":"2018-11-07T14:47:48.000+00:00",
+  "publishing_app":"specialist-publisher",
+  "rendering_app":"finder-frontend",
+  "schema_name":"finder",
+  "search_user_need_document_supertype":"government",
+  "title":"News and communications",
+  "updated_at":"2018-11-07T14:47:48.000+00:00",
+  "user_journey_document_supertype":"finding",
+  "withdrawn_notice":{
+
+  },
+  "publishing_request_id":"11295-1516787799.287-10.3.3.1-400",
```

Comment:
Can remove this I think - it might be misleading to leave it in.  If we can't remove it, we'll need to set it to something fake/findable